apps/snitch_core/lib/core/data/model/tax/tax_configuration.ex

```diff
@@ -24,4 +24,26 @@ defmodule Snitch.Data.Model.TaxConfig do
   def get(id) do
     QH.get(TaxConfig, id, Repo)
   end
+
+  @doc """
+  Gets the tax config set for the store. Since a single row
+  is set, it always returns the same config.
+  """
+  def get_default() do
+    [config] = Repo.all(TaxConfig)
```

Comment:
What if there is no TaxConfig struct? It'll throw a match error : [config] = [ ]

Reply:
Already suggested this change :smirk: